source/manual/aws-iam-key-rotation.html.md

```diff
@@ -5,93 +5,131 @@ layout: manual_layout
 type: learn
 section: AWS
 owner_slack: "#govuk-developers"
-last_reviewed_on: 2019-10-31
+last_reviewed_on: 2020-01-17
 review_in: 6 months
 ---
 
 ## Overview
 
-GOV.UK uses [AWS IAM](https://docs.aws.amazon.com/en_pv/IAM/latest/UserGuide/id_users.html)
-accounts to manage applications access to our AWS infrastructure. The
-[Access Keys](https://docs.aws.amazon.com/en_pv/IAM/latest/UserGuide/id_credentials_access-keys.html)
-associated with these accounts need to be [rotated](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_RotateAccessKey)
-every 3 months.
+GOV.UK uses [AWS IAM] accounts to manage applications access to our AWS
+infrastructure. The [Access Keys] associated with these accounts need to be
+[rotated] every 3 months.
 
-## How to rotate Access Keys
+Rotating AWS IAM keys requires permissions which are available with the `admin`,
+`internal-admin`, `platformhealth-poweruser` or `poweruser` roles. If you can't
+assume one of these roles, you won't be able to rotate keys.
 
-> **Note**
->
-> Rotating AWS IAM keys requires permissions which are available with the
-> `admin`, `internal-admin`, `platformhealth-poweruser` or `poweruser` roles.
-> If you can't assume one of these roles, you won't be able to rotate keys.
+## How to rotate Access Keys
 
-Log in to the [IAM Management Console](https://console.aws.amazon.com/iam/home?region=eu-west-1#/users)
-for the environment you want to rotate keys for and you'll see a list of users
-and the age of their access keys.
+Log in to the [IAM Management Console] in the environment you want to rotate
+keys for and you'll see a list of users and the age of their access keys.
 
 ### 1. Find users access key
 
-Click on the user whose key you want to rotate. Click on `Security Credentials`. Make a note of the Access Key ID.
+- Click on the user whose key you want to rotate.
+- Click on the `Security Credentials` tab.
 
-### 2. Find where the key is used
+Users should always only have one access key. The only time a user should have
+two keys is when we are in the process of rotating the old one. For this reason,
+AWS allow users to have a maximum of two Access Keys at the same time.
 
-Enter [govuk-secrets/puppet](https://github.com/alphagov/govuk-secrets/tree/master/puppet) in your machine.
-Run one of the [Puppet AWS common actions](https://github.com/alphagov/govuk-secrets/tree/master/puppet_aws#common-actions) and search for the `Access Key ID` in the result.
+If this user has no keys, consider deleting it if it's not needed.
```

Comment:
It's unlikely that someone reading this documentation will come across this scenario, because they should be looking at a user which has an old key.

Maybe say something like "If the user has a single key, that appears not to be in use, consider removing the user"?

Reply:
Yes, I thought about that too, but I would leave it anyway because if someone is in the process of rotating an old key and wats to have a look at the other keys and users at the same time, at least they know what they should do.

Reply:
I know we've talked about that, but I'd be wary that this might be a bit beyond the scope of the document. I'm also unsure how risky this is, as I haven't tried deleting users in AWS which are in use, so I'm not sure how easy it is to break things.

Reply:
Ok, let's take it out entirely for the moment. We can then revisit this in a user-specific document or somewhere else if needed.